c10/core/TensorImpl.cpp

```diff
@@ -939,14 +946,20 @@ void TensorImpl::set_sizes_and_strides(
       extra_meta_->storage_offset_ = storage_offset_;
     }
   }
-  extra_meta_->sizes_ = sizes;
+  clone_symvec(sizes, extra_meta_->sizes_);
   extra_meta_->strides_ = strides;
+  // this segfaults
+  // clone_symvec(strides, extra_meta_->strides_);
```

Comment:
Hopefully these don't segfault anymore

Reply:
yea, the segfault was mainly coming from cases where the symint wasn't symbolic and i was trying to force a symbolic pointer out of it.  but the resize thing wasn't good either.  in any case it seems stable now